src/exchanges/bybit/orderbooks/local-orderbook.ts

```diff
@@ -7,7 +7,7 @@ const OrderBooks = new OrderBooksStore({
 });
 
 // connect to a websocket and relay orderbook events to handlers
-const ws = new WebsocketClient({ testnet: false, market: 'inverse' });
+const ws = new WebsocketClient({ testnet: false, market: 'v5' });
```

Comment:
did you test whether the sample works after changing the market to v5? the events might be different